Fitness/Views/ProBioPageViewController.swift

```diff
@@ -0,0 +1,301 @@
+//
+//  ProBioPageViewController.swift
+//  
+//
+//  Created by Cameron Hamidi on 2/28/19.
+//
+
+import UIKit
+
+class ProBioPageViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout {
+
+    var backButton: UIButton!
+    
+    var pro: ProBio!
+    
+    var scrollView: UIScrollView!
+    
+    var profilePicContainerView: UIView!
+    var profilePicView: UIImageView!
+    var nameLabel: UILabel!
+    
+    var bioTitleLabel: UILabel!
+    var bioTextView: UITextView!
+    
+    var bioExpertiseDivider: UIView!
+    
+    var expertiseTitleLabel: UILabel!
+    var expertiseTextView: UITextView!
+    
+    var expertiseRoutinesDivider: UIView!
+    
+    var routinesTitleLabel: UILabel!
+    var routinesView: AllProRoutinesView!
+    
+    var linksCollectionView: UICollectionView!
+    var linksTitleLabel: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        view.backgroundColor = .white
+        
+        scrollView = UIScrollView()
+        scrollView.backgroundColor = .white
+        view.addSubview(scrollView)
+        
+        profilePicContainerView = UIView()
+        profilePicContainerView.backgroundColor = .white
+        scrollView.addSubview(profilePicContainerView)
+        
+        profilePicView = UIImageView()
+        profilePicView.contentMode = .scaleAspectFill
+        profilePicView.clipsToBounds = true
+        profilePicContainerView.addSubview(profilePicView)
+        
+        nameLabel = UILabel()
+        nameLabel.layer.zPosition = 1
+        nameLabel.textColor = .white
+        nameLabel.font = ._36MontserratBold
+        nameLabel.textAlignment = .center
+        nameLabel.sizeToFit()
+        scrollView.addSubview(nameLabel)
+        
+        bioTitleLabel = UILabel()
+        bioTitleLabel.textColor = .black
+        bioTitleLabel.font = ._20MontserratSemiBold
+        bioTitleLabel.textAlignment = .center
+        bioTitleLabel.sizeToFit()
+        bioTitleLabel.numberOfLines = 0
+        scrollView.addSubview(bioTitleLabel)
+        
+        bioTextView = UITextView()
+        bioTextView.textColor = UIColor.colorFromCode(0x222222)
+        bioTextView.font = ._14MontserratRegular
+        bioTextView.sizeToFit()
+        bioTextView.isScrollEnabled = false
+        bioTextView.textAlignment = .center
+        bioTextView.isSelectable = false
+        scrollView.addSubview(bioTextView)
+
+        bioExpertiseDivider = UIView()
+        bioExpertiseDivider.backgroundColor = UIColor.colorFromCode(0xE5ECED)
```

Comment:
Bring out all of these color codes to UIColor+Extension file, similar to fonts.

Reply:
Yup, will make these constants in UIColor